Frontend/CO.CDP.OrganisationApp.Tests/Registration/OrganisationIdentificationTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿using CO.CDP.OrganisationApp.Pages.Registration;
+using CO.CDP.OrganisationApp.Tests.Common;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace CO.CDP.OrganisationApp.Tests.Registration
+{
+    public class OrganisationIdentificationModelTests
+    {
+        private readonly OrganisationIdentificationModel _pageModel;
+        private readonly Session _session;
+        private readonly DefaultHttpContext _httpContext;
+        private readonly Mock<ILogger<OrganisationIdentificationModel>> _loggerMock;
+
+        public OrganisationIdentificationModelTests()
+        {
+            _loggerMock = new Mock<ILogger<OrganisationIdentificationModel>>();
+
+            _httpContext = new DefaultHttpContext
+            {
+                Session = new MockHttpSession()
+            };
+            var httpContextAccessor = new HttpContextAccessor { HttpContext = _httpContext };
+
+            _session = new Session(httpContextAccessor);
+
+            _pageModel = new OrganisationIdentificationModel(_loggerMock.Object, _session);
+        }
+
+        [Fact]
+        public void ValidIdentificationNumber_ShouldStoreInSession()
+        {
+            // Arrange
```

Comment:
Preferably we don't need comments unless its need explicit explaining in our code.

Reply:
@dharmverma I have removed any comments from the code.